projects/spectator/src/lib/spectator/host-component.ts

```diff
@@ -0,0 +1,6 @@
+import { Component } from '@angular/core';
```

Comment:
should be in spectator-with-host-folder

Reply:
Should we? Because it is the default Host for Spectator.

Reply:
`createTestComponentFactory` doesn't use it.